src/spikeinterface/sorters/launcher.py

```diff
@@ -26,14 +26,14 @@
     joblib=dict(n_jobs=-1, backend="loky"),
     processpoolexecutor=dict(max_workers=2, mp_context=None),
     dask=dict(client=None),
-    slurm=dict(tmp_script_folder=None, cpus_per_task=1, mem="1G"),
+    slurm={"tmp_script_folder": None, "sbatch_executable_path": "sbatch", "cpus-per-task": 1, "mem": "1G"},
 )
 
 
 _implemented_engine = list(_default_engine_kwargs.keys())
 
 
-def run_sorter_jobs(job_list, engine="loop", engine_kwargs={}, return_output=False):
+def run_sorter_jobs(job_list, engine="loop", engine_kwargs=None, return_output=False):
```

Comment:
🙌

Reply:
honestly we really should discuss this more!
I think I really do not like it.

Reply:
I'll make a thread on this / cc on the existing discussion to continue the chat from last week, I think this is also a good example to consider. If your happy to leave as-is for now I'll link it on that issue and we can change to later with the rest of such cases.

Reply:
#3226